File: node/hello.js
// om namah shivay

const express = require('express');

const app = express();

app.use('/', (req, res) => {
    res.json({ 'message': 'Hello from Node!' });
});

const port = process.env.PORT || 8080;
app.listen(port, () => {
    console.log(`Node server listening on port ${port}`);
});

File: go/hello.go
// om namah shivay

package main

import (
	"net/http"
	"log"
	"encoding/json"
)

func main() {
	
	http.HandleFunc("/", hello)
	
	log.Print("Go server listening on port 8080")
	log.Fatal(http.ListenAndServe(":8080", nil))

}

func hello(w http.ResponseWriter, r *http.Request) {

	if r.URL.Path != "/" {
		http.NotFound(w, r)
		return
	}

	dictionary := make(map[string]string)
	dictionary["message"] = "Hello from Go!"

	json, _ := json.Marshal(dictionary)

	w.Header().Set("Content-Type", "application/json")
	w.Write(json)

}
